### PR TITLE
[minigraph.py]: Parse mux cable table

### PR DESCRIPTION
**Pull request by theasianpianist**
Signed-off-by: Lawrence Lee <lawlee@microsoft.com>

**- Why I did it**

**- How I did it**
- Parse device IPv6 loopback addresses when available
- Add MUX_CABLE table to config DB:

```
'MUX_CABLE': {
	'Ethernet4': {
    	'state': 'auto',
        'server_ipv4': '10.10.10.1/32',
        'server_ipv6': 'fe80::0001/128'
	}
}
```
**- How to verify it**

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by theasianpianist** [comment]
retest mellanox please

**Comment by theasianpianist** [comment]
retest mellanox please

**Comment by theasianpianist** [comment]
retest mellanox please

**Review by prsunny — APPROVED** [review]
lgtm.. @tahmed-dev , @vdahiya12

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -1273,10 +1282,24 @@ def get_tunnel_entries(tunnel_intfs, lo_intfs, hostname):
...
     return tunnels
 
+def get_mux_cable_entries(mux_cable_ports, neighbors, devices):
+    mux_cable_table = {}
+
+    for intf in mux_cable_ports:
+        if intf in neighbors:
+            entry = {}
+            neighbor = neighbors[intf]['name']
+            entry['state'] = 'auto'
```
[the remark this answers is not in the document]
Comment (theasianpianist) [reply src/sonic-config-engine/minigraph.py:1292]: Yes, we can use a different state if that's preferred.

[the remark this answers is not in the document]
Comment (vdahiya12) [reply src/sonic-config-engine/minigraph.py:1292]: @tahmed-dev can you suggest what should the default state be ?

[the remark this answers is not in the document]
Comment (tahmed-dev) [reply src/sonic-config-engine/minigraph.py:1292]: @theasianpianist , @vdahiya12 `auto` seems about right.

File: src/sonic-config-engine/tests/test_minigraph_case.py [hunk 2]
```diff
@@ -203,3 +220,24 @@ def test_minigraph_tunnel_table(self):
...
+                'server_ipv4': '10.10.10.2/32',
+                'server_ipv6': 'fe80::0002/128'
+            }
+        }
+
+        output = self.run_script(argument)
+        self.assertEqual(
+            utils.to_dict(output.strip()),
+            expected_table
+        )
```
Comment (tahmed-dev) [inline src/sonic-config-engine/tests/test_minigraph_case.py:243]: EOL?

**Review by tahmed-dev — APPROVED** [review]

**Review by vdahiya12 — APPROVED** [review]

**Pull request merged**